modules/authentication/client-react/access/jwt/index.jsx

```diff
@@ -4,15 +4,16 @@ import { withApollo } from 'react-apollo';
 import PropTypes from 'prop-types';
 
 import { getItem, setItem, removeItem } from '@module/core-common/clientStorage';
+import { CURRENT_USER_QUERY } from '@module/user-client-react';
```

Comment:
It contradicts our conception here, that auth module should be independent of concrete user module

Reply:
Done